webdriver-spec.html

```diff
@@ -4398,7 +4443,7 @@
 </ol>
 </section> <!-- /Execute Async Script -->
 </section> <!-- /Executing Script -->
-
+</section> <!-- /Document Handling -->
```

Comment:
This is in the wrong place.  Since your new chapter _Document Handling_ comes directly after _Element Interaction_, it should be closed **before** _Executing Script_ begins.